build-system/release-tagger/make-release.js

```diff
@@ -187,19 +191,36 @@ function _createBody(head, base, prs) {
 }
 
 /**
- * Main function
+ * Fetch release if exists
+ * @param {string} head
+ * @return {!Promise}
+ */
+async function fetchRelease(head) {
+  try {
+    return await getRelease(head);
+  } catch {}
+}
```

Comment:
naming nit: At first glance, the words `fetch` and `get` seem like synonyms. In order to convey the _"if it exists"_ difference, perhaps call this function `maybeGetRelease()` so it's clear that `getRelease()` assumes the release exists while this function does not?

Reply:
Done